src/components/Status/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Status.css';
+
+const Status = (props) => {
+  return (
+    <div className={`StatusSignal StatusIs${props.status}`} />
```

Comment:
hey @StephenAraka  instead of passing in a string 'On' that is even case sensitive. because i tried 'on' and it refused. Why not use this? it is a better implementation 
```suggestion
<div>
      {props.status ? 
        (<div className={'StatusSignal StatusIsOn'} />):
        (<div className={'StatusSignal StatusIsOff'} />)
      }
    </div>```

Reply:
You just pass a Boolean of true and one of false. it does the magic too

Reply:
> You just pass a Boolean of true and one of false. it does the magic too

@Mubangizi  ... Cool, lemme do this... Thanks for the feedback

Reply:
> hey @StephenAraka instead of passing in a string 'On' that is even case sensitive. because i tried 'on' and it refused. Why not use this? it is a better implementation

@Mubangizi I have made the changes... Changed it to this instead...👇. (To maintain a single line)